src/types/sdk.ts

```diff
@@ -23,29 +23,48 @@ export interface SdkError {
   message: string
 }
 
-export interface SdkOptions {
+export type ServerRegions = 'US' | 'EU' | 'CA'
+
+interface UndocumentedOptions {
```

Comment:
I would choose a different name for these options. Maybe `FunctionalConfigurations`?

Reply:
Did you mean this?

```suggestion
interface FunctionalConfigurations {
```

Reply:
Yes 👍